### PR TITLE
project one

### PR DESCRIPTION
**Pull request by elyami600**
yamil.perez1@lc.cuny.edu

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
```
Comment (lehmanclass) [inline index.html]: can you put all your code in a `project_one` folder? this file should show up in the diff as `project_one/index.html`

File: about.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">about</a></li>
+
+                <li><a href="index.html">Gmail</a></li>
+                <li><a href="about.html">Images</a></li>
+
+            </ul>
+        </nav>
+    </div>
+    <h1 id="grading-element-one-consequence">about page</h1>
```
Comment (lehmanclass) [inline about.html]: can you change it to `id=#grading-element-two-consequence`? otherwise works as expected

File: index.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        <br>
+
+        <div id="grading-element-three-consequence" hidden >
+
+            Today will be amazing day
+
+        </div>
+
+
+        <button id="grading-element-one">Click Me!</button>
```
Comment (lehmanclass) [inline index.html]: works as expected

File: index.html [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+            <input class="inputHide" value="value" name="TestThree" type="text" required>
+            <input type="submit"/>
+         </form>
+
+        <button onclick="Onvisible()">Luck</button>
+
+        <br>
+
+        <div id="grading-element-three-consequence" hidden >
```
Comment (lehmanclass) [inline index.html]: stays `hidden` even after i click `#grading-element-three`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+                <span style="color:yellow ">O</span>
+                <span style="color: cornflowerblue">G</span>
+                <span style="color:green ">L</span>
+                <span style="color: red">E</span>
+            </h1>
+        </div>
+
+
+
+        <form  action="http://localhost:3000/form" name="grading-element-four" method="POST">
```
Comment (lehmanclass) [inline index.html]: works as expected

File: index.html [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+                <li><a href="about.html">about</a></li>
+
+            </ul>
+        </nav>
+    </div>
+
+    <div class="main-body">
+
+        <div class="main-title">
+            <h1 id="grading-element-five">
```
Comment (lehmanclass) [inline index.html]: does not change color when i hover

**Comment by lehmanclass** [comment]
needs a few fixes. Please share any extra credit you do in a a `README.md` file.

File: index.html (hunk 4 above)
Comment (elyami600) [reply index.html]: I fixed the error

File: index.html (hunk 6 above)
Comment (elyami600) [reply index.html]: I fixed the error

**Pull request closed without merging**